src/pelz/main.c

```diff
@@ -5,9 +5,8 @@
 #include <getopt.h>
 #include <stdlib.h>
 
-#include "key_table.h"
-#include "pelz_service.h"
 #include "pelz_log.h"
+#include "pelz_io.h"
 
 #include "sgx_urts.h"
```

Comment:
I don't think this main should need either ```sgx_urts.h``` or ```pelz_enclave.h```.

Reply:
Removed extra includes